src/main/kotlin/no/nav/yrkesskade/saksbehandling/graphql/client/pdl/IPdlClient.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.yrkesskade.saksbehandling.graphql.client.pdl
+
+import com.expediagroup.graphql.generated.HentIdenter
+import com.expediagroup.graphql.generated.enums.IdentGruppe
+import com.expediagroup.graphql.generated.hentperson.Person
+
+interface IPdlClient {
+    fun hentAktorId(fodselsnummer: String): String?
```

Comment:
Regner med at du har kopiert alle pdl-kall fra melding-mottak, og det er jo greit. Til info så er de ulike kallene tilpasset for å hente minimalt med data til visse caser, så mulig vi må utvide (eller minke) datautplukket for tilpassing til saksbehandling

Reply:
Stemmer! Jeg har tilpasset hentPerson til saksbehandling case.